app/views/categories/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<% provide(:title, @category.name) %>
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
```

Comment:
Maybe you mean category_info?

Reply:
i haven't added any css so this is just a placeholder, ill change the name